### ISSUE TITLE
Q: azurerm_app_service_custom_hostname_binding multiple count with same app_service_name 

### ISSUE THREAD

**Issue by kevinneufeld** [terraform-providers/terraform-provider-azurerm#1968]
Not able to apply count on `azurerm_app_service_custom_hostname_binding` with multiple hostnames that targets the same `app_service_name` without a `409 conflict` failure. 

```shell
* azurerm_app_service_custom_hostname_binding.test.0: web.AppsClient#CreateOrUpdateHostNameBinding: Failure sending request: StatusCode=409 -- Original Error: autorest/azure: Service returned an error. Status=<nil> <nil>
2018/09/21 15:20:19 [ERROR] root: eval: *terraform.EvalSequence, err: 1 error(s) occurred:
``` 
I can keep cycling through plan and apply to get through the `conflict` errors and complete the plan.

Is there a way to delay a resource between counts or maybe a dependency of some sort? 
```hcl
locals {
   hosts = ["test01.somedomain.com", "test02.somedomain.com"]
}

resource "azurerm_app_service_custom_hostname_binding" "test" {
  count               = 2
  hostname            = "${local.hosts[count.index]}"
  app_service_name    = "test-app-service"
  resource_group_name = "test-app-rg"

  depends_on = []
}
```

**Comment by tombuildsstuff** [comment]
hey @kevinneufeld

Thanks for opening this issue :)
shoul
> Is there a way to delay a resource between counts or maybe a dependency of some sort?

There is - using `depends_on` and separate resources - however this should work with the code sample you've provided above - as such I've taken a look into this and have opened #1970 which includes a fix for this.

Thanks!

**Comment by kevinneufeld** [comment]
Thanks @tombuildsstuff.

**Comment by tombuildsstuff** [comment]
hey @kevinneufeld 

Just to let you know that this fix has now landed in v1.16 of the AzureRM Provider (here's the full changelog](https://github.com/terraform-providers/terraform-provider-azurerm/blob/v1.16.0/CHANGELOG.md)). You can upgrade to to this version by specifying it in your provider block:

```
provider "azurerm" {
  version = "=1.16.0"
}
```

... and then running `terraform init -upgrade` :)

Thanks!
